src/components/Button/src/Button.vue

```diff
@@ -120,7 +141,7 @@ export default {
 		size: {
 			type: String,
 			default: undefined,
-			validator: (size) => ['small', 'medium', 'large'].includes(size),
+			validator: (size) => ['small', 'medium', 'large', 'expandable'].includes(size),
```

Comment:
can you please document in the Button's styleguide how the `expandable` size is different from the other sizes and when/why it would be used?

Reply:
Okay, after looking through the CSS i see that `expandable` is basically the same as `medium` except that `medium` sets `height: 48px` where `expandable` sets `min-height: 48px` hence the height becomes expandable.

Rather than adding an `expandable` value, what do you think about adding `min-small`, `min-medium`, and `min-large` values? where those styles are the same as their `small`, `medium`, and `large` counterparts except that they set `min-height` instead of `height`?